locales/en-US/app.ftl

```diff
@@ -1169,6 +1169,14 @@ BottomBox--assembly-code-not-available-title = Assembly code not available
 BottomBox--assembly-code-not-available-text =
     See <a>issue #4520</a> for supported scenarios and planned improvements.
 
+# The toggle button for making the bottom box fullscreen.
+BottomBox--hide-fullscreen =
+    .title = Minimize the source and assembly views to no longer be fullscreen.
```

Comment:
```suggestion
    .title = Restore view
```

Or `Exit fullscreen`.

* Minimize and restore are very different behaviors
* Tooltips shouldn't have a closing period